sov-modules/sov-state/src/backend.rs

```diff
@@ -0,0 +1,45 @@
+use crate::{
+    storage::{StorageKey, StorageValue},
+    Prefix, Storage,
+};
+use sovereign_sdk::serial::{Decode, Encode};
+use std::marker::PhantomData;
+
+#[derive(Debug)]
+pub(crate) struct Backend<K, V, S> {
+    _phantom: (PhantomData<K>, PhantomData<V>),
+    storage: S,
+    // Every instance of the `Backend` contains a unique prefix.
```

Comment:
(nit, non-blocking): It would be nice to make this a doc comment unless there's a reason not to

Reply:
done